readme.md

```diff
@@ -1,3 +1,70 @@
 
-Instructions will be released as they are developed
+Instructions will be released as they are developed.
+
+## Assignment Submission Summary
+Assignments are submitted in two parts:
+1. Push your changes to GitHub.
+2. Verify that your changes are showing on GitHub, then copy and paste the URL into the assignment text box on D2L.
+
+## Git and GitHub Summary
+Words in `<brackets>` are just placeholders - do not type the `<>`.
+### To start an assignment
+1. To start your assignment, click on the GitHub Classroom link in D2L and click "accept this assignment". Refresh the page to get your unique assignment URL.
+2. On your computer, run the command `git clone <url>`, where `<url>` is your assignment URL (do not type the `<>`).
+
+### Working on an assignment
+3. Make sure you are working in your assignment directory, either through Git Bash or VS Code's integrated terminal.
+4. Pull changes from GitHub, just in case:
+   ```
+   git pull
+   ```
+5. Make edits to your text file(s). Don't forget to save changes!
+6. Add/commit/push changes:
+   1. First, add the file(s) you want to commit to the staging area:
+        ```
+        git add <filename>
+        ```
+   2. Then, commit all the files with a message describing the changes you made:
+        ```
+        git commit -m "<message>"
+        ```
+   3. Push all your commits to GitHub
+        ```
+        git push
+        ```
+> You can add/commit/push as many times as you like up until you submit your assignment
+
+### Final submission
+7. When you are happy with the state of your code, add/commit/push one last time and go to your assignment repository URL. You can find this by clicking on the "GitHub Classroom link" for the assignment in D2L, or from the GitHub homepage.
+8. Double check that the last commit on GitHub is the last one you made on your computer.
+9. Copy and paste the assignment repository URL into the D2L assignment text box. This is your signal to your instructor that you are done the assignment. It should look something like:
+    ```
+    https://github.com/MRU-F22-CS1/assign-####-<github username>
+    ```
+    > There may be slight variations in how your instructor named your assignment repo
+
+## Troubleshooting
+For more details, you can also refer back to the [exercise 1](exercise1/intro_to_git_v1.md) instructions.
+
+### Changes aren't showing on GitHub.com
+Make sure that you have done all of the following:
+
+ 1. **Save the file** from your text editor.
+ 2. `git add <filename>`, where `<filename>` is the name of the file you are adding (e.g. `git add README.md`).
+ 3. `git commit -m "commit message"`, where `commit message` is a *useful* note about why you are committing the file.
+ 4. `git push`.
+
+ If you still aren't seeing your changes on GitHub, make sure that you don't accidentally have multiple copies of your repo on your local computer. It's easy to accidentally edit one copy while git bash is open in another.
+
```

Comment:
And if you **still** still aren't seeing your changes, get in touch with an IA or your instructor.